orphaned test fails due to usr-is-merged package

**What happened:**
- *Brief description of Bug*
With the update of debootstrap to version 1.0.127 the package usr-is-merged is installed by debootstrap as a requirement. The package usr-is-merged is detected by the orphaned test as orphaned therefore the test fails.

**Debug Logs:**
```
FAILED base/test/test_orphaned.py::test_orphaned - AssertionError: Found orphaned packages: ['usr-is-merged']
```

--- a/features/base/exec.config
+++ b/features/base/exec.config
@@ -5,3 +5,8 @@ set -Eeuo pipefail
 ln -sf /etc/dpkg/origins/gardenlinux /etc/dpkg/origins/default
 
 chmod u-s /bin/umount /bin/mount
+
+# Issue #1137
+# Mark package as manual installed to pass the orphaned test
+# The package is installed and required by debootstrap 1.0.127
+apt-mark manual usr-is-merged

